d3b_dff_cli/modules/validation/check_manifest.py

```diff
@@ -43,14 +45,30 @@ def validate_row(row, rules):
                 if col == "file_size" and row.get("file_format", "").lower() in ["fastq", "bam", "cram"]:
                     greater_than_value = consequence.get("greater_than")
                     if greater_than_value:
-                        try:
-                            file_size_in_gb = float(row.get(col, 0)) / (1024 * 1024 * 1024)  # Convert to GB
-                            if file_size_in_gb <= float(greater_than_value.rstrip(" GB")):
-                                error_messages.append(f"Warning: *{col}* less than {greater_than_value}")
-                        except ValueError:
-                            error_messages.append(f"*{col}* is not a valid numeric value")
+                        experiment = row.get("experiment_strategy", "").lower()
+                        if experiment in ["wgs", "wxs", "wes"]:
+                            greater_than_value = "1 GB"
+                            minum_value = 1_000_000_000 # WGS/WXS should be greater than 1G
```

Comment:
similar to above comment. We are storing great than variables in the json file. Should we also store the minum values there?

Reply:
> similar to above comment. We are storing great than variables in the json file. Should we also store the minum values there?

here the `minum_value` is just a kind of converted name for `greater_than_value`, changing from GB to Bytes.